test/user.test.ts

```diff
@@ -1,9 +1,10 @@
 import { describe, expect } from '@jest/globals'
+import type { AuthConfig } from '../src/models/Common/Auth'
 import { Skolengo } from '../src/index'
 import './common'
 
 const SKOLENGO_TOKENSET = process.env.SKOLENGO_TOKENSET
-const describeAuthenticated = !!SKOLENGO_TOKENSET ? describe : describe.skip
+const describeAuthenticated = (SKOLENGO_TOKENSET !== null && SKOLENGO_TOKENSET !== '') ? describe : describe.skip
```

Comment:
Pourquoi avoir fait cette modification ? :thinking:
Une variable d'environnement ne peut jamais être `null`. Lorsqu'elle est vide, elle est soit `undefined` soit `""`.
L'opérateur *double-bang* permet de savoir si la variable d'environnement contient une valeur ou non.
```ts
console.log(!!null)    // false
console.log(!!"")    // false
console.log(!!"hello world")    //true
```

Reply:
`SKOLENGO_TOKENSET !== undefined` suffit

Reply:
D'après mes tests, ceci devrait fonctionner :
```suggestion
const describeAuthenticated = SKOLENGO_TOKENSET !== undefined ? describe : describe.skip
```